libs/internal/include/launchdarkly/data_model/flag.hpp

```diff
@@ -88,7 +88,7 @@ struct Flag {
     std::vector<Target> targets;
     std::vector<Target> contextTargets;
     std::vector<Rule> rules;
-    std::optional<std::uint64_t> offVariation;
+    std::optional<std::int64_t> offVariation;
```

Comment:
Our JSON specification notes that the minimum values are 0, so I used `uint64_t` - but that's not the case in practice, as the test harness may send negative values.